Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -111,8 +156,10 @@ import BraintreeCore
 
             self.apiClient.post("", parameters: graphQLParameters, httpType: .graphQLAPI) { body, _, error in
                 if error != nil {
+                    let jsonResponse: BTJSON? = (error as? NSError)?.userInfo[BTCoreConstants.jsonResponseBodyKey] as? BTJSON
```

Comment:
Since we are passing through the error now we can do an `if let error` on the line above to avoid the double unwrapping here. If let will check that the value is `!= nil`.

Reply:
Do you mean something like this: 
```
if let error = error as? NSError {
      let jsonResponse: BTJSON? = error.userInfo[BTCoreConstants.jsonResponseBodyKey] as? BTJSON
      ...
}
```
Just to clarify, we are only passing through the error msg only in the case of venmo payment context service validation errors.